git-open.plugin.zsh

```diff
@@ -0,0 +1,6 @@
+# This plugin is MIT licensed to match the git-open license.
+#
+# Make git-open easy to install and keep up to date if you're using a
+# ZSH framework like Zgen or Antigen
+
+  export PATH=${PATH}:$(dirname $0)
```

Comment:
I think you should move the dirname path before existing $PATH, like the omz npm plugin:

https://github.com/robbyrussell/oh-my-zsh/blob/286c3e5e28b651c3f629e6aaba589a1cf56da4d8/plugins/npm/npm.plugin.zsh
and the omz pyenv plugin:
https://github.com/robbyrussell/oh-my-zsh/blob/83cf8dc16f51babbb0193c5b97e568739c1f40de/plugins/pyenv/pyenv.plugin.zsh#L19

If people use this plugin via omz/antigen/etc. they probably expect it to be the priority "git open", where they may have a different git-open elsewhere in their $PATH

So just swap to `export PATH="$(dirname $0):${PATH}"`